middleware/downloadcsv.js

```diff
@@ -1,18 +1,55 @@
 const { parse, transforms: { unwind } } = require("json2csv")
 
+const get_qcode = (row) => {
+  if (row.answers.options && row.answers.options.q_code) {
+    return row.answers.options.q_code
+  }
+  if (row.answers.options && row.answers.options.qCode) {
+    return row.answers.options.qCode
+  }
+  if (row.answers.q_code) {
+    return row.answers.q_code
+  }
+  if (row.answers.qCode) {
+    return row.answers.qCode
+  }
+}
+
```

Comment:
Hi Josh, just a syntax suggestion - if you're happy this function could be simplified using the optional chaining operator (`?.`):

```suggestion
const get_qcode = (row) =>
    row.answers?.options?.q_code
    || row.answers?.options?.qCode
    || row.answers?.q_code
    || row.answers?.qCode;
```

Reply:
Tom, I get this when pasting your code in - Uncaught SyntaxError: Unexpected token .

Reply:
Ah, no worries - just tried it and it seems you need node v14 (or to use babel - it's used in the author codebase but I guess that gets preprocessed by babel via create-react-app). I admit, I only tried it in the console to start with ;-)

Reply:
I'm looking at upgrading extract to node 14

Reply:
That's certainly a neat and readable implementation, is it worth updating to node 14 to facilitate this?